pdns/dnsdist-web.cc

```diff
@@ -205,6 +228,8 @@ static void connectionThread(int sock, ComboAddress remote, string password)
         resp.headers["Content-Type"] = "text/css";
       else if(parts.back() == "js")
         resp.headers["Content-Type"] = "application/javascript";
```

Comment:
DNSSec on Bind is broke. 2 different distros and hours on #bind and never found a solution. It work with one or 2 zone but as soon as you cram an feed it 5 or 6 new domains for DNSsec, it just dies.
powerdns fixed all of that and a number of other anomalies first shot, and only took a couple of minutes to configure.
This is a very well maintained project.